PerilEndorsements.py

```diff
@@ -0,0 +1,102 @@
+import requests
+import json
```

Comment:
See the order of imports: https://www.python.org/dev/peps/pep-0008/#imports

1) Standard library imports.
2) Related third party imports.
3) Local application/library specific imports.

So your script should begin with the four lines:
```
#!/usr/bin/env python
import json

import requests
```

Reply:
If you write this in an IDE, you can have it automatically order these for you.

Reply:
I'm not sure if this is true.